libphal/phal_exception.H

```diff
@@ -24,6 +24,7 @@ enum ERR_TYPE {
 	SBE_CMD_FAILED,
 	SBE_CMD_TIMEOUT,
 	SBE_FFDC_NO_DATA,
+	SBE_BOOT_SEEPROMS_BAD,
```

Comment:
**Commit message:**

> Adding support for recovery action REIPL_BKP_BSEEPROM/
REIPL_UPD_BSEEPROM (REIPL/Update backup seeprom)

Please update what needs to be done for the REIPL_BKP_BSEEPROM and REIPL_UPD_BSEEPROM recovery action in the commit message.